### PR TITLE
Update display functions and documentation

### PR DESCRIPTION
**Pull request by AxelHamburch**
This PR optimizes the feedback for the operator by displaying two more messages and also adding another button function. In addition, optimizations are made and the documentation will be updated.

List of changes:

- New display message function: "update_button_fault()" = Notifies you of an invalid pulse count
- New display message function: update_wallet_fault() = Notifies you of an invalid wallet setup
- Added calls for the new display messages and also tested for the Waveshare 2in13v2 display.
- Add display test function (9 times button pushes)
- The messages.py file has been extended for German and French. Adapted to German, the French tests remain in English.
- All other display drivers updated. Since I don't have the displays and therefore can't physically test them, I copied and adjusted the "def update_nocoin_screen():" and all rows were set to the x-axis of the first row.
- Mulitiligual for papiruszero2in.py
- Preset for the PaPyRus display to avoid error messages
- Documentation updates

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -229,6 +233,98 @@ def button_pushed():
         logger.warning("ATM shutdown (5 times button)")
         os.system("sudo shutdown -h now")
 
+    if config.PUSHES == 9:
```
Comment (lightisfaster) [inline app.py:236]: was already fully implemented in the display align test

**Review by lightisfaster — CHANGES_REQUESTED** [review]
Thanks for this PR.
There is a test displayalign with similar funktion. Complete this in the app.py inspired by the displayalign test or delete the test funktion from the app.py. Alternativ we can try to call the test from the app.py. Cheers Ingo

File: app.py (hunk 1 above)
Comment (AxelHamburch) [reply app.py:237]: Unfortunately I saw that too late. Now I've brought all the ads into the call in app.py as well.

**Comment by AxelHamburch** [comment]
The display indication function is now complete. The code in the app.py is manageable and the possibility of calling it up via the button also has its charm. I think that should work now.

**Pull request merged**